demosplan/DemosPlanCoreBundle/Application/DemosPlanKernel.php

```diff
@@ -104,10 +104,12 @@ public function registerBundles(): iterable
 
         // Register all addons
         $addonRegistry = new AddonRegistry();
-        $addonRegistry->configureAutoloading();
 
         foreach ($addonRegistry->getAllAddons() as $addonData) {
-            yield new $addonData['manifest']['entry']($addonData['enabled']);
+            $class = $addonData['manifest']['entry'];
+            if (class_exists($class, true)) {
```

Comment:
Hm. I can agree with you but I kind of fear this will lead to debugging sessions where we wonder why an addon doesn't work and forgot that it may just be silently not being loaded.